system-addon/lib/ActivityStream.jsm

```diff
@@ -130,6 +130,10 @@ const PREFS_CONFIG = new Map([
     title: "Boolean flag that decides whether or not to show the topstories disclaimer.",
     value: true
   }],
+  ["section.highlights.excludePocket", {
```

Comment:
Not sure about what namespace we want for the "sub-option" prefs

Reply:
I guess it matches "section.highlights.collapsed" although we also have "browser.newtabpage.activity-stream.collapseTopSites"…

Let's keep it next to the other highlights pref. And perhaps rename it to "includePocket" as that's what the UI would expose.